docs/zh/latest/plugins/google-cloud-logging.md

```diff
@@ -114,25 +118,27 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
 
 ## 测试插件
 
-* 向配置 `google-cloud-logging` 插件的路由发送请求
+你可以通过以下命令向 APISIX 发出请求：
 
 ```shell
-$ curl -i http://127.0.0.1:9080/hello
+curl -i http://127.0.0.1:9080/hello
+```
+
+```
 HTTP/1.1 200 OK
 ...
 hello, world
 ```
 
-* 登录谷歌云日志服务，查看日志
-
-[Google Cloud Logging Service](https://console.cloud.google.com/logs/viewer)
+访问成功后，你可以登录[谷歌云日志服务](https://console.cloud.google.com/logs/viewer)查看相关日志。
```

Comment:
The previous text was in English but suddenly changed to Chinese here. It is recommended to keep it unified.